### PR TITLE
Rename babily to babel-minify

### PR DESCRIPTION
**Pull request by benbrandt**
Addressing issue #1307 

Mostly updated links and package/preset references. I left the actual names in the blog posts alone because I wasn't sure if those should be changed. I also left the REPL alone since it seemed like the standalone package isn't ready yet.

Let me know if there should be any more changes!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ If you encounter an `Liquid Exception: Included file '_includes/.../packages/...
 npm test
 ```
 
-This will lint markdown documents inside both the website and the imported repositories (like babel, babili, etc).
+This will lint markdown documents inside both the website and the imported repositories (like babel, babel/minify, etc).
```
Comment (existentialism) [inline README.md]: Nit: babel-minify

File: _posts/2016-08-26-babili.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ categories: announcements
 share_text: "Babili (babel-minify)"
 ---
 
-We released [Babili](https://github.com/babel/babili) as beta (0.0.1) a few days ago under an MIT license!
+We released [Babili](https://github.com/babel/minify) as beta (0.0.1) a few days ago under an MIT license!
```
Comment (existentialism) [inline _posts/2016-08-26-babili.md]: Think it's fine to leave the blog posts alone for historical purposes, the repo redirects anyway.

**Comment by existentialism** [comment]
@benbrandt thanks! just a few nits

**Comment by benbrandt** [comment]
Ok let me know if you have any other changes!

**Comment by existentialism** [comment]
@benbrandt thanks!

**Pull request merged**